sympy/physics/control/lti.py

```diff
@@ -0,0 +1,553 @@
+    >>> tf3
+    TransferFunction((p - 1)*(p + 3), (p - 1)*(p + 5), p)
+
+    Using Float or Integer (or other constants) as numerator and denominator.
+
+    >>> tf4 = TransferFunction(1/2, 4, s)
+    >>> tf4.num
+    0.500000000000000
+    >>> tf4.den
+    4
+    >>> tf4.var
+    s
+
+    References
+    ==========
+
+    Joao P. Hespanha, Linear Systems Theory. 2009.
+
+    """
+    def __new__(cls, num, den, var):
+        num, den = _sympify(num), _sympify(den)
+
+        if not isinstance(var, Symbol):
+            raise TypeError("Variable input must be a Symbol.")
+        if den == 0:
+            raise ValueError("TransferFunction can't have a zero denominator.")
+
+        if (((isinstance(num, Expr) and num.has(Symbol)) or num.is_number) and
+            ((isinstance(den, Expr) and den.has(Symbol)) or den.is_number)):
+                obj = super(TransferFunction, cls).__new__(cls, num, den, var)
+                obj._num = num
+                obj._den = den
+                obj._var = var
+                return obj
+        else:
+            raise TypeError("Unsupported type for numerator or denominator of TransferFunction.")
+
+    @property
+    def num(self):
+        """Returns the Numerator of the Transfer Function.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction(s**2 + p*s + 3, s - 4, s)
+        >>> G1.num
+        p*s + s**2 + 3
+        >>> G2 = TransferFunction((p + 5)*(p - 3), (p - 3)*(p + 1), p)
+        >>> G2.num
+        (p - 3)*(p + 5)
+
+        """
+        return self._num
+
+    @property
+    def den(self):
+        """Returns the Denominator of the Transfer Function.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction(s + 4, p**3 - 2*p + 4, s)
+        >>> G1.den
+        p**3 - 2*p + 4
+        >>> G2 = TransferFunction(3, 4, s)
+        >>> G2.den
+        4
+
+        """
+        return self._den
+
+    @property
+    def var(self):
+        """Returns the variable that is used to anchor the symbols.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction(p**2 + 2*p + 4, p - 6, p)
+        >>> G1.var
+        p
+        >>> G2 = TransferFunction(0, s - 5, s)
+        >>> G2.var
+        s
+
+        """
+        return self._var
+
+    @property
+    def bound_symbols(self):
+        """Returns only variables that are dummy variables.
+
+        Examples
+        ========
+
+        >>> from sympy.abc import s, p, l
+        >>> from sympy.physics.control.lti import TransferFunction
+        >>> G1 = TransferFunction(s + l, s - 2*p, s)
+        >>> G1.bound_symbols
+        [s]
```

Comment:
Why is p not in this list?

Reply:
I think it's only for returning the anchor variables as we're only considering them to be bound

Reply:
That reminds me if we're having a `bound_symbols` method for returning bound symbols, should we also have a `free_symbols` method returning the rest of the symbols?

Reply:
Yes!

Reply:
okay, I will define another method to return free symbols.